src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -32,16 +35,22 @@ class Application extends BaseApplication
     private KernelInterface $kernel;
     private bool $commandsRegistered = false;
     private array $registrationErrors = [];
+    private readonly bool $debug;
```

Comment:
Does it really belong to the component rather than framework 
-bundle ? Or is it because the patched logic is in this class and not the other?

Reply:
I don't understand, this is in the framework-bundle.

Reply:
Trapped by github mobile's path shortening :) sorry